### DISCUSSION TITLE
PLEASE HELP ME FIND THIS "NOTEPAD" EXTENSION

### DISCUSSION THREAD

**Discussion by knana1662** [community/community#51012] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I found a particular extension to try out and tried searching for the "notepad" or notebook extension in vscode extension but cannot find it. Please help me with the name and a link to it if anyone uses it using the image below. Thank You
<img width="286" alt="primary-sidebar1" src="https://user-images.githubusercontent.com/109052744/227725254-4d7a9936-a431-4109-8a57-f9e53183f354.png">

**Comment by Anmol-Baranwal** [comment]
Based on the image you provided, it looks like the extension you are looking for is called "Jupyter" or "Jupyter Notebook".

Alternatively, you can also find the Jupyter extension on the VS Code Marketplace website using this link: https://marketplace.visualstudio.com/items?itemName=ms-toolsai.jupyter.

**Reply by knana1662** [reply]
I am talking about any extension with the Notepad"notes" and "archive" sections from above.
